includes/definitions/discovery/avocent.yaml

```diff
@@ -1,6 +1,11 @@
-mib: PM-MIB
+mib: PM-MIB:ACS-MIB
 modules:
     sensors:
+        pre-cache:
+            data:
+                -
+                    oid:
+                        - acsSerialPortTableName
```

Comment:
I suggest caching ACS-MIB::acsSerialPortTableName instead.  Reduces issues with mib search.  (You'll have to update the usage below too.

Reply:
Done